Cargo.toml

```diff
@@ -1,7 +1,12 @@
 [package]
 name = "rustybuzz"
-version = "0.20.0"
-authors = ["Yevhenii Reizner <razrfalcon@gmail.com>"]
+version = "0.20.1"
+authors = [
+    "Caleb Maclennan <caleb@alerque.com>",
+    "Laurenz Stampfl <laurmaedje@gmail.com>",
```

Comment:
Email is from a different Laurenz 👀 Mine is laurenz.stampfl@gmail.com

Reply:
Yikes. Also not the first time I've conflated you!